### ISSUE TITLE
Expected 2-3 arguments, but got 1.

### ISSUE THREAD

**Issue by jjangga0214** [ardatan/graphql-import#468]
Info: 1.0.0-beta.1

**Bug**

```ts
const schema = await importSchema('graphql/schema/schema.graphql')
```

When using typescript, this throws an error `Expected 2-3 arguments, but got 1.ts(2554)`.
I believe this is a type bug, as additional arguments are intended to be optional, aren't they?

**Workaround**

Just add an empty object as second argument.

```ts
const schema = await importSchema('graphql/schema/schema.graphql', {})
```

**Comment by ardatan** [comment]
Fixed in `v1.0.0-beta.2`.

**Comment by ardatan** [comment]
Available in 1.0.0!
